test/jdk/java/net/DatagramSocket/DatagramSocketMulticasting.java

```diff
@@ -57,11 +60,17 @@
 import static java.net.StandardSocketOptions.SO_REUSEADDR;
 import static jdk.test.lib.NetworkConfiguration.isSameInterface;
 
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Is this needed?

Reply:
I don't think so.

One test instance per test class is useful for when having multiple test methods that want to share state in instance fields. Here, we have a single test method and no shared state.

Find details at https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle
> If you would prefer that JUnit Jupiter execute all test methods on the same test instance, annotate your test class with `@TestInstance(Lifecycle.PER_CLASS)`. When using this mode, a new test instance will be created once per test class. Thus, if your test methods rely on state stored in instance variables, you may need to reset that state in `@BeforeEach` or `@AfterEach` methods.

Reply:
It isn't, I'll clean that up